pkg/controller/syncclusterdeployment/syncclusterdeployment_controller.go

```diff
@@ -0,0 +1,445 @@
+/*
```

Comment:
remotemachineset may be a better name for the controller/package

Reply:
+1 to this. The sync name gets confusing when all controllers sync as their main action.